pkg/get/tools.go

```diff
@@ -1,6 +1,25 @@
 package get
 
-func MakeTools() []Tool {
+import "strings"
+
+func (t Tools) Len() int { return len(t) }
```

Comment:
Given that you're sorting the tools themselves, can you also add a unit test please?

Just create some fake data perhaps with three values and show that the order is as expected.

Reply:
Thanks for your comment, done.